README.md

```diff
@@ -40,23 +40,37 @@ Details on each supported algorithm can be found in the [docs/algorithms](https:
 
 <!--- OQS_TEMPLATE_FRAGMENT_LIST_KEXS_START -->
 - **BIKE**: BIKE-L1, BIKE-L3
+- **BIKE**: BIKE-L1, BIKE-L3
+- **Classic McEliece**: Classic-McEliece-348864†, Classic-McEliece-348864f†, Classic-McEliece-460896†, Classic-McEliece-460896f†, Classic-McEliece-6688128†, Classic-McEliece-6688128f†, Classic-McEliece-6960119†, Classic-McEliece-6960119f†, Classic-McEliece-8192128†, Classic-McEliece-8192128f†
```

Comment:
Duplicate lines?

Reply:
Hm, I didn't the portion of the update_docs script that updates the README.md. For now I'm pushing a manually fixed version of the README, but I'll have to look into why duplicate lines are appearing now.

Reply:
Oh, I see the issue. It's on my system. It looks like a bunch of 'family 2.yml's were made for some reason. Either way, the README should be ok now.